### ISSUE TITLE
ROCKET, MINIROCKET, MultiRocket - To scale or not to scale?

### ISSUE THREAD

**Issue by Huii** [ChangWeiTan/MultiRocket#3]
Hello,

I'm currently in the process of running a comparative study on my data, including the models ROCKET, MINIROCKET and MultiRocket. I would like to know a bit more about how and when to scale. In a [previous issue](https://github.com/angus924/minirocket/issues/1), @angus924 wrote the following: " Unlike Rocket, MiniRocket does not require the input time series to be normalised. In fact, it may hurt performance. By default, we do not normalise input time series for MiniRocket."

Following this advice, I'm currently not normalizing the signals for MINIROCKET but do so for ROCKET. For MulitRocket, I also do not plan to include any normalizing. Thus, my current strategy is as follows:
 1. ROCKET - Scaling with[ StandardScaler()](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html) from scikit-learn
 2. MINIROCKET - No Scaling
 3. MultiRocket - No Scaling

Do you have any comments on this? Does my strategy allow for a fair comparison? Thank you very much in advance.

**Comment by ChangWeiTan** [comment]
@Huii, Yes scaling for minirocket seems to hurt the performance. For multirocket, if you are using minirocket kernels, then it is better to not scale. If you are using rocket kernels, then you can scale your data although I have not tested rocket without scaling the data.

**Comment by Huii** [comment]
Thank you very much for your reply. 

So far, my (preliminary) results on different signals and different transformations of those signals always yield the best performance for ROCKET (scaled). MINIROCKET  (unscaled) has the second best performance and MultiRocket sometimes performs worse and sometimes better than MINIROCKET if I scale the data first before classification. I will give it another try without scaling anything and report back here.

**Comment by ChangWeiTan** [comment]
Ok. for the default MultiRocket, it is better to not scale as it uses Minirocket kernels. If you change to Rocket kernels, then it is better to scale the data.
